### ISSUE TITLE
ERC20 Collection bug: ERC20 doesn't being collected with `Collect with Fees` mode.

### ISSUE THREAD

**Issue by streetcrypto7** [rubykube/peatio#2294]
ERC20 deposit should be collected with `Collect with Fees`  first, however, peatio tries with normal `collect` mode and therefore, skip the collection due to insufficient funds from deposit wallet.

**Comment by streetcrypto7** [comment]
Same issue with this one

![캡처](https://user-images.githubusercontent.com/52137633/62422458-14dcd800-b6ee-11e9-8d51-786b86106654.PNG)

**Comment by ysv** [comment]
@streetcrypto7 Actually it's known issue. And its' more or less design problem since we can't delay message publish on Bunny RabbitMQ.
We are working on solving this issue.  
For now we advice to Collect with Fees first time and then try Collect if Collect with Fees will not succeed for the first time.

**Comment by streetcrypto7** [comment]
Yeah, but I would like to know how to collect them without clicking `Collect with Fees` button 😂
